src/lib/image.py

```diff
@@ -0,0 +1,55 @@
+from PIL import Image, ImageEnhance
+import numpy as np
+
+
+def reset(rgb_color, pixel_height, pixel_width):
+    """
+    Resets the image with the specified RGB color.
+
+    Args:
+            rgb_color (tuple): A tuple containing the RGB values of the color.
+            pixel_height (int): The height of the image in pixels.
+            pixel_width (int): The width of the image in pixels.
+
+    Returns:
+            numpy.ndarray: A numpy array representing the reset image.
+    """
+    r, g, b = rgb_color
+    return np.full([pixel_height, pixel_width, 3], [b, g, r], np.uint8)
```

Comment:
3 is the number of colors?   Maybe switch to len(rgb_color).

Reply:
Think this is OK to be this way.  because the LED strips is always based on RGB, they don't have any support for any other color types.  If we ever had to support something other than RGB probably have to rethink some of these methods on how it handles colors fundamentally.